account_check_deposit/report/report_checkdeposit.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright (C) 2014-2019 Akretion (www.akretion.com)
+    @author: Alexis de Lattre <alexis.delattre@akretion.com>
+    The licence is in the file __manifest__.py
+-->
+<odoo>
+    <template id="report_checkdeposit">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="web.internal_layout">
```

Comment:
I'll suggest to use the external layout for this document, as it has to be given to the bank. It's a way to "communicate" to external partner.
`<t t-call="web.external_layout">`

Reply:
@alexis-via 😄

Reply:
You are certainly right ! I'm not a qweb expert, I have more experience on py3o.